src/main/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountExtractServiceImpl.java

```diff
@@ -0,0 +1,691 @@
+                if (attSecurity != null && (attSecurity.isHide() || attSecurity.isMask() || attSecurity.isPartialMask())
+                        && !allowedAchSecureFields.contains(propertyName)) {
+                    // Replace potentially-sensitive placeholders with an empty string.
+                    replacement = KFSConstants.EMPTY_STRING;
+                } else {
+                    // Replace the placeholder with the property value, or with an empty string if null or invalid.
+                    try {
+                        Object propertyValue = ObjectPropertyUtils.getPropertyValue(achAccount, propertyName);
+                        replacement = ObjectUtils.isNotNull(propertyValue) ? propertyValue.toString() : KFSConstants.EMPTY_STRING;
+                        // If a values finder is defined, use the label from the matching key/value pair instead.
+                        if (attDefinition.getControl() != null && StringUtils.isNotBlank(attDefinition.getControl().getValuesFinderClass())) {
+                            KeyValuesFinder valuesFinder = (KeyValuesFinder) Class.forName(attDefinition.getControl().getValuesFinderClass()).newInstance();
+                            String key = replacement;
+                            replacement = valuesFinder.getKeyLabel(key);
+                            // If the key is in the label, then remove it from the label.
+                            if (attDefinition.getControl().getIncludeKeyInLabel() != null
+                                    && attDefinition.getControl().getIncludeKeyInLabel().booleanValue()) {
+                                // Check for key-and-dash or key-in-parentheses, and remove them if found.
+                                // (Former can come from BO values finders, latter is only for custom values finders that append the keys as such.)
+                                String keyAndDashPrefix = key + " - ";
+                                String keyInParenSuffix = " (" + key + ")";
+                                replacement = replacement.startsWith(keyAndDashPrefix) ? StringUtils.substringAfter(replacement, keyAndDashPrefix)
+                                        : (replacement.endsWith(keyInParenSuffix)
+                                                ? StringUtils.substringBeforeLast(replacement, keyInParenSuffix) : replacement);
+                            }
+                        }
+                        // Because of the way that Matcher.appendReplacement() works, escape the special replacement characters accordingly.
+                        if (replacement.indexOf('\\') != -1) {
+                            replacement = replacement.replace("\\", "\\\\");
+                        }
+                        if (replacement.indexOf('$') != -1) {
+                            replacement = replacement.replace("$", "\\$");
+                        }
+                        
+                    } catch (ClassNotFoundException e) {

```

Comment:
Looks like since we're developing for [Java 7 and above, you can simplify these catch statements into one](http://stackoverflow.com/questions/3495926/can-i-catch-multiple-java-exceptions-in-the-same-catch-clause):
```java
} catch (ClassNotFoundException | IllegalAccessException | InstantiationException | RuntimeException e) {
  replacement = KFSConstants.EMPTY_STRING;
}
```